packages/trpc/server/createContext.ts

```diff
@@ -143,5 +143,3 @@ export const createContext = async ({ req, res }: CreateContextOptions, sessionG
     res,
   };
 };
-
-export type Context = trpc.inferAsyncReturnType<typeof createContextInner>;
```

Comment:
you were only using this type in the `initTRPC` call, which now accepts just passing the function directly so no need to keep this extra type laying around - personal preference perhaps..